libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount+Internal.h

```diff
@@ -24,12 +24,10 @@
 
 #import "SFUserAccount.h"
 
-// The temporary user identity
-static NSString * const SFUserAccountManagerTemporaryUserAccountUserId = @"TEMP_USER_ID";
-static NSString * const SFUserAccountManagerTemporaryUserAccountOrgId = @"TEMP_ORG_ID";
 
 @interface SFUserAccount ()
 
 @property (nonatomic, readwrite, getter = isUserDeleted) BOOL userDeleted;
+@property (nonatomic, readwrite, getter = isUserLoggingOut) BOOL userLoggingOut;
```

Comment:
Minor: should this just be `loggingOut`/`isLoggingOut`?  The "user" part seems redundant.